lib/facility_factory.rb

```diff
@@ -0,0 +1,62 @@
+class FacilityFactory
+  attr_reader :new_facilities
+              :dmv_office_locations
+
+  def initialize
+    @new_facilities = []
+  end
+   
+  
+  def checks_location(location_data)
+      dmv_office_locations(location_data)
+  end
+  
+  def dmv_office_locations(location_data)
```

Comment:
This is an awfully large method. A good refactor would be to see what you can pull out into helper methods to make things more dynamic. What if you needed to pull data from 20 more offices? 

So think about having one method that is responsible for creating new `Facility` objects. Then build helper methods that can format things like name, address, phone_number and use those inside your main method that is creating the new objects. Just remember, you have to test EVERY method you create. 
Currently you are not testing this method. You are only testing `checks_location`.